docs/user-guide/install.md

```diff
@@ -1,7 +1,15 @@
-Iter8 can be installed and configured to watch resources either in a single namespace (namespace scoped) or in the whole cluster (cluster scoped). 
+---
+template: main.html
+---
+
+# Install options
+
+By default, Iter8 uses [BadgerDB]() to store metrics from A/B/n and performance tests. BadgerDB is not suitable for production use. To install for production, use another database. Database options are listed [here](metrics_store.md).
 
 ## Install with `helm`
 
+Iter8 can be installed and configured to watch resources either in a single namespace (namespace scoped) or in the whole cluster (cluster scoped). 
```

Comment:
```suggestion
Iter8 can be installed and configured to watch resources either in a single namespace (namespace-scoped) or in the whole cluster (cluster-scoped). 
```

Not sure if we set this pattern in the past but wondering if it should be like this.